content/docs/configuration.md

```diff
@@ -83,7 +83,30 @@ synced_files:
     dest: .
 ```
 
+## RPM spec file not in upstream repository
```

Comment:
This is also closely tight to the use case when a spec file needs to be processed before it's usable.

Reply:
Sorry, I don't follow, what do you want to write it here.
Something like:

```yaml
actions:
   post_upstream_clone:
       - wget <SPEC file>
       - wget <All SOURCES and PATCHES>

```

Reply:
don't worry, I can send a PR after yours is merged to not overload this one